packages/lesswrong/components/comments/CommentsNode.jsx

```diff
@@ -38,11 +38,21 @@ class CommentsNode extends PureComponent {
     super(props);
     this.state = {
       hover: false,
-      collapsed: props && props.comment && (props.comment.baseScore < KARMA_COLLAPSE_THRESHOLD || props.comment.deleted),
+      collapsed: this.isCollapsed(),
+      toggleUncollapsed: false,
       finishedScroll: false,
     };
   }
 
+  isCollapsed = () => {
+    const { comment, nestingLevel, totalCommentCount } = this.props
+    return (
+      comment.deleted ||
+      comment.baseScore < KARMA_COLLAPSE_THRESHOLD ||
+      (totalCommentCount > 25 && nestingLevel === 3)
```

Comment:
These parameters (25 and 3) seem important and tweakable enough to be worth hoisting out to constants at the top of the file, and maybe all the way out as far as into settings.json.